packages/geometry/src/matrix4.ts

```diff
@@ -384,7 +384,7 @@ export function makeOrthographic(
 
   const x = (right + left) * w;
   const y = (top + bottom) * h;
-  const z = (far + near) / d;
+  const z = (far + near) * d;
```

Comment:
That would make a difference...

Reply:
Yeah... definitely explained a lot of issues I had been seeing initially...